### PR TITLE
LPS-183631 | Test Fix | Master

### PR DESCRIPTION
**Pull request by victorhcunha**
[LPS-183631](https://issues.liferay.com/browse/LPS-183631)

### CODE REVIEW

**Comment by victorhcunha** [comment]
ci:test:sf

**Review by Luizcarlosqueiroz — APPROVED** [review]
LGTM!

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/IFrame.path [hunk 1]
```diff
@@ -62,7 +62,7 @@
 </tr>
 <tr>
 	<td>DIALOG_BOX</td>
-	<td>//div[contains(@class,'modal-header')]/h3</td>
+	<td>//div[contains(@class,'modal-title')]</td>
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/IFrame.path:65]: @victorhcunha merged in https://github.com/liferay-bpm-qa/liferay-portal/pull/826. Can we remove this commit? Thanks!

Comment (victorhcunha) [reply portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/IFrame.path]: @patriciajeanperez commit removed!

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**